examples/siro_sandbox/README.md

```diff
@@ -1,165 +1,136 @@
-# Sandbox Tool 
-
-![siro_sandbox_screenshot](https://user-images.githubusercontent.com/6557808/230213487-f4812c2f-ec7f-4d68-9bbe-0b65687f769b.png)
-
-# Overview
-This is a 3D interactive GUI app for testing various pieces of SIRo, e.g. rearrangement episode datasets, Fetch and Spot robots, humanoids (controllers, animation, skinning), trained agent policies, batch rendering and other visualization.
-
-# Known Issues
-* The tool is not very stable in the `SIRo` branch due to rapid iteration in various parts of the codebase. See Snapshots section below for best results.
-* The skinned humanoid doesn't render correctly. If you prefer, you can hide it with `--hide-humanoid-in-gui`.
-* When using HSSD scenes (see below), the app has bad runtime perf on older Macbooks (2021 is fine; 2019 is bad). See "Workaround for poor runtime perf on slower machines".
-
-# Snapshots with examples of running the tool
-See [SIRo Sandbox Snapshots Google Doc](https://docs.google.com/document/d/1cvKuXXE2cKchi-C_O7GGVFZ5x0QU7J9gHTIETzpVKJU/edit#). The tool is not very stable in the `SIRo` branch due to rapid iteration in various parts of the codebase. This doc describes well-tested sets of commits across our repos (Habitat-lab, Habitat-sim, fphab, floorplanner). This doc also gives example commands to run the tool.
-
-# Example commands
-### GUI-controlled humanoid and learned-policy-controlled Spot
-
-* To launch GUI-controlled humanoid and random-policy-controlled (initialized with random weights) Spot, in HSSD run:
-```
+# Sandbox Tool
+
+This is a 3D interactive GUI app capable of running human-in-the-loop (HITL) evaluation in Habitat. It can be driven either locally using mouse and keyboard, or remotely using a Unity-based VR client.
+
+<p align="center">
+  <img src="../../res/img/hitl_tool.gif">
+</p>
+
+## Table of Contents
```

Comment:
Again, I'm guessing this isn't the final readme we want to open-source, so I won't nitpick too much. With that said, at a high level, what we want to cover in the introduction:
* this is an example app using the HITL framework
* brief overview of what this app does
* how should users build their own apps? E.g. copy-paste these files? Inherit from these classes? Branch habitat-lab and modify these files in place? (correct answer is mostly the first option here: copy-paste these files).
* alternately, how should users customize this example app? E.g. adding a new app state.


Also, we eventually want to get away from command-line args for all the app options. Once they are moved to e.g. a hydra config yaml, they can be documented there instead of in this readme.